js/ui/calendar/ui.calendar.selection.strategy.js

```diff
@@ -0,0 +1,227 @@
+import { noop } from '../../core/utils/common';
+import Class from '../../core/class';
+import dateUtils from '../../core/utils/date';
+
+const abstract = Class.abstract;
+
+const DAY_INTERVAL = 86400000;
+
+const CalendarSelectionStrategy = Class.inherit({
```

Comment:
please, let's move different strategies to different files how it's already done in
[selection strategies](https://github.com/DevExpress/DevExtreme/tree/23_1/js/ui/selection)
[datebox strategies](https://github.com/DevExpress/DevExtreme/tree/23_1/js/ui/date_box)
and others.
It's very difficult to search for an overided method when all strategies are places in a single file.

Reply:
yeah, I saw how it's done there, but in this case there are only 200 lines for all strategies, as for me it's more handfull to keep them in one file. But ok, I mooved them to separate files